File: python/plus_minus.py
#!/bin/python3

import math
import os
import random
import re
import sys


#
# Complete the 'plusMinus' function below.
#
# The function accepts INTEGER_ARRAY arr as parameter.
#

def plusMinus(arr):
    positives = 0
    negatives = 0
    neutros = 0
    tamanho = len(arr)
    for value in arr:
        if value > 0:
            positives = positives + 1
        elif value < 0:
            negatives = negatives + 1
        else:
            neutros = neutros + 1
    print(round(positives / tamanho, 6))
    print(round(negatives / tamanho, 6))
    print(round(neutros / tamanho, 6))


if __name__ == '__main__':
    n = int(input().strip())

    arr = list(map(int, input().rstrip().split()))

    plusMinus(arr)

File: c#/plusMinus.cs
using System.CodeDom.Compiler;
using System.Collections.Generic;
using System.Collections;
using System.ComponentModel;
using System.Diagnostics.CodeAnalysis;
using System.Globalization;
using System.IO;
using System.Linq;
using System.Reflection;
using System.Runtime.Serialization;
using System.Text.RegularExpressions;
using System.Text;
using System;

class Result
{

    /*
     * Complete the 'plusMinus' function below.
     *
     * The function accepts INTEGER_ARRAY arr as parameter.
     */

    public static void plusMinus(List<int> arr)
    {
        Int32 length = arr.Count;
        int countPositives = 0;
        int countNegatives = 0;
        int countNeutros = 0;
        foreach (int element in arr){
            if(element > 0)
            {
                countPositives++;
            }
            else if(element < 0)
            {
                countNegatives++;
            }
            else 
            {
                countNeutros++;    
            }
        }
        double positives = (double)countPositives / length;
        double negatives = (double)countNegatives / length;
        double neutros = (double)countNeutros / length;

        Console.WriteLine(positives.ToString("F6"));
        Console.WriteLine(negatives.ToString("F6"));
        Console.WriteLine(neutros.ToString("F6"));   
    }

}

class Solution
{
    public static void Main(string[] args)
    {
        int n = Convert.ToInt32(Console.ReadLine().Trim());

        List<int> arr = Console.ReadLine().TrimEnd().Split(' ').ToList().Select(arrTemp => Convert.ToInt32(arrTemp)).ToList();

        Result.plusMinus(arr);
    }
}